HIP-0008.md

```diff
@@ -115,6 +138,30 @@ that keys are derived from a series of 4-byte indexes. Indexes lower than `0x7ff
 are derived using non-hardened derivation, which is required if an algorithm
 only has access to the public key.
 
+## The Meta-Account Key
+
+To generate the `metaxpub` from the master private key, we propose a BIP44 derivation
+path beginning with `purpose = 5353'`. This is a hardened derivation and so the
+actual value is `0x800014e9`. Following that, `coinType` and `accountIndex`
```

Comment:
The first three nodes in the Ledger bip32 path (`m'/44'/5353'/`) can actually not be changed. 

https://github.com/LedgerHQ/app-handshake/pull/4/commits/19d63b7ddc530ab96010afa5280f621b2e16e045

So we should move the meta-account key to some account index. Probably adding `0x40000000` to whatever hardened account index (minimum `0x80000000`) is already being used by the wallet. This will cut the account space in half (from 2 billion to 1 billion) but makes it accessible from hardware wallets. We should still test with a Ledger to make sure we are allowed to derive accounts this high up.